bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableBufferedMutator.java

```diff
@@ -157,21 +168,51 @@ public void testInvalidPut() throws Exception {
   @SuppressWarnings("unchecked")
   @Test
   public void testZeroWorkers() throws Exception {
-    when(client.mutateRowAsync(any(MutateRowRequest.class))).thenReturn(future);
+    when(mockClient.mutateRowAsync(any(MutateRowRequest.class))).thenReturn(mockFuture);
     Configuration config = new Configuration(false);
     config.set(BigtableOptionsFactory.BIGTABLE_ASYNC_MUTATOR_COUNT_KEY, "0");
     try (BigtableBufferedMutator underTest = createMutator(config)) {
       underTest.mutate(SIMPLE_PUT);
-      verify(client, times(1)).mutateRowAsync(any(MutateRowRequest.class));
+      verify(mockClient, times(1)).mutateRowAsync(any(MutateRowRequest.class));
       Assert.assertTrue(underTest.hasInflightRequests());
       completeCall();
       Assert.assertFalse(underTest.hasInflightRequests());
     }
   }
 
+  @Test
```

Comment:
Might be beneficial to have a test, or enhance this test, to cover the case where multiple mutates / multiple mutateRows calls get sent to test the resetting of BulkMutation state.